extensions/amp-story/1.0/amp-story.js

```diff
@@ -1623,6 +1623,9 @@ export class AmpStory extends AMP.BaseElement {
               mutation => mutation.attributeName === 'open')) {
             this.storeService_.dispatch(Action.TOGGLE_SIDEBAR,
                 this.sidebar_.hasAttribute('open'));
+          } if (mutationsList.some(
```

Comment:
nit: This will probably be obsoleted by @gmajoulet's comment above, but I strongly prefer putting multiple `if`s on different lines (i.e. put 1 or more newlines between the closing brace and the `if`), to visually distinguish that this is _not_ an `if/else if` case.

Reply:
This makes sense! I will definitely keep this in mind for the future for readability.